.jvmopts

```diff
@@ -0,0 +1,2 @@
+-Xms1g
```

Comment:
I don't think that's a related change?

Reply:
Alas it is, well you are free to remove the lower end, it seems the additional build is enough to break the max heap, a snippet from [here](https://github.com/chris-twiner/frameless/actions/runs/6298705747/job/17098045684):

``` 
java.lang.OutOfMemoryError: GC overhead limit exceeded
	at scala.reflect.internal.Symbols$Symbol.createValueParameterSymbol(Symbols.scala:1361)
	at scala.reflect.internal.Symbols$Symbol.newTermSymbol(Symbols.scala:1375)
	at scala.reflect.internal.Symbols$TermSymbol.cloneSymbolImpl(Symbols.scala:2909)
```

Reply:
Issue is that it must be only for CI, not on every developer env

Reply:
I've swapped it out  for CI only in dfd3ebc